### PR TITLE
add custom github actions

### PR DESCRIPTION
**Pull request by shibuiwilliam**
# What
- Add go test action using the custom action, https://github.com/marketplace/actions/record-build-and-test-results-action .
- Fix main branch to master branch, since we are not using main branch in the repository.

### CODE REVIEW

File: .github/workflows/go_custom_action.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      - name: Install dependencies
+        run: |
+          go install github.com/jstemmer/go-junit-report@latest
+      - name: Test
+        run: cd go && go test -v ./... 2>&1 | go-junit-report -set-exit-code > report.xml
+      - name: Record
+        uses: launchableinc/record-build-and-test-results-action@v1.0.0
```
Comment (Konboi) [inline .github/workflows/go_custom_action.yml:29]: So simple

File: .github/workflows/bazel.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ name: bazel
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
   push:
-    branches: [main]
+    branches: [master]
```
Comment (Konboi) [inline .github/workflows/bazel.yml:9]: BTW, I want to change the base branch from master to main

**Review by Konboi — APPROVED** [review]
💯

File: .github/workflows/bazel.yml (hunk 2 above)
Comment (shibuiwilliam) [reply .github/workflows/bazel.yml:9]: Sounds awesome!

**Pull request merged**